README.md

```diff
@@ -514,6 +515,20 @@ let app =
     ]
 ```
 
+### redirectTo
+
+`redirectTo` uses a 302 response code to redirect the browser to the specified path.
+
+#### Example:
+
+```fsharp
+let app = 
+    choose [
+        route "/" >=> redirectTo "/"
```

Comment:
`route "/" >=> redirectTo "/"` is maybe a weird example. Perhaps `route "/" >=> redirectTo "/foo"` would be better.